### PR TITLE
Pull request to add robots.txt and sitemap.xml files (v1)

### PR DESCRIPTION
**Pull request by alexisrc1**

### CODE REVIEW

File: examples/official-site/robots.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+User-agent: *
+Disallow: /examples/authentification/  # Block the folder "authentification" to avoid blocking by password
+Disallow: /examples/hash_password.sql  # Block access to specific sensitive SQL files like this
+Disallow: /Dockerfile  # Block access to Dockerfile
```
Comment (lovasoa) [inline examples/official-site/robots.txt:4]: Why ? https://sql.datapage.app/Dockerfile

File: examples/official-site/robots.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+User-agent: *
+Disallow: /examples/authentification/  # Block the folder "authentification" to avoid blocking by password
+Disallow: /examples/hash_password.sql  # Block access to specific sensitive SQL files like this
```
Comment (lovasoa) [inline examples/official-site/robots.txt:3]: Why ? This is a useful page, it is not password protected: https://sql.datapage.app/examples/hash_password.sql

File: examples/official-site/robots.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+User-agent: *
+Disallow: /examples/authentification/  # Block the folder "authentification" to avoid blocking by password
```
Comment (lovasoa) [inline examples/official-site/robots.txt:2]: Maybe do not block the entire authentication example ? We still want it referenced. Ideally change the auth example so the password protected page has a different url, that you can block

File: examples/official-site/sitemap.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://sql.datapage.app/</loc>
+    <lastmod>2024-10-10</lastmod>
```
Comment (lovasoa) [inline examples/official-site/sitemap.xml:5]: Is it ok to hardcode the date like this ? Won't the prevent re-crawling when we update the page ?

File: examples/official-site/sitemap.xml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  <url>
+    <loc>https://sql.datapage.app/documentation.sql</loc>
+    <lastmod>2024-10-10</lastmod>
+    <priority>0.80</priority>
+  </url>
+  <url>
+    <loc>https://sql.datapage.app/your-first-sql-website/</loc>
+    <lastmod>2024-10-10</lastmod>
+    <priority>0.80</priority>
+  </url>
```
Comment (lovasoa) [inline examples/official-site/sitemap.xml:22]: Is it ok to have such a little subset of the site in the sitemap ? Won't it penalize other pages ?

We should probably either not have a sitemap, or have one that is generated dynamically in sql

**Review by lovasoa — CHANGES_REQUESTED** [review]

File: examples/official-site/robots.txt (hunk 2 above)
Comment (alexisrc1) [reply examples/official-site/robots.txt:3]: which one is ?

File: examples/official-site/robots.txt (hunk 1 above)
Comment (alexisrc1) [reply examples/official-site/robots.txt:4]: I was told to limit the number of files that need to be crawled so I figure this one is not useful for Google

File: examples/official-site/sitemap.xml (hunk 4 above)
Comment (alexisrc1) [reply examples/official-site/sitemap.xml:5]: if it remains hard coded for too long it is an issue, for this first try it is ok

**Pull request merged**